jinad/api/endpoints/logs.py

```diff
@@ -0,0 +1,41 @@
+import uuid
+import time
+
+from fastapi import APIRouter
+
+from fastapi.responses import StreamingResponse
+
+router = APIRouter()
+
+
+@router.get(
+    path='/log/{id}',
+    summary='Stream logs from folder where fluentd stores',
+)
+def _log(
```

Comment:
This should work with `async def` too. `def` executes the code in a separate thread-pool

Reply:
Isn't it good though?

Reply:
Do not we explicitly want this to work in a different `ThreadPool`? Like this every stream will have its own thread?